src/main/java/gregtech/api/metatileentity/implementations/GT_MetaTileEntity_MultiBlockBase.java

```diff
@@ -1931,6 +1931,11 @@ public void getWailaBody(ItemStack itemStack, List<String> currentTip, IWailaDat
         super.getWailaBody(itemStack, currentTip, accessor, config);
     }
 
+    public final void getOriginalWailaBody(ItemStack itemStack, List<String> currentTip, IWailaDataAccessor accessor,
```

Comment:
```suggestion
    public final void getMTEWailaBody(ItemStack itemStack, List<String> currentTip, IWailaDataAccessor accessor,
```
to make it clear which "original" this refers to in case of deeper inheritance